### PR TITLE
Removed ViewPump library

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/BackpackDemoApplication.kt [hunk 1]
```diff
@@ -38,12 +35,6 @@ class BackpackDemoApplication : Application() {
 
     private lateinit var instance: BackpackDemoApplication
 
-    var highlight = false
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/BackpackDemoApplication.kt:41]: This flag was used to use Backpack coverage indicators in the app

File: app/src/main/java/net/skyscanner/backpack/demo/BpkBaseActivity.kt [hunk 2]
```diff
@@ -47,10 +45,6 @@ open class BpkBaseActivity : AppCompatActivity() {
     ShakeListener(this::onShaked)
   }
 
-  override fun attachBaseContext(newBase: Context) {
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/BpkBaseActivity.kt:50]: We were performing the wrapping in order to indications to work

File: app/src/main/java/net/skyscanner/backpack/demo/ComponentDetailActivity.kt [hunk 3]
```diff
@@ -101,10 +101,6 @@ class ComponentDetailActivity : BpkBaseActivity() {
         component_detail_container.layoutDirection = View.LAYOUT_DIRECTION_LTR
       }
     }
-    if (keyCode == KeyEvent.KEYCODE_H && event?.isShiftPressed == true) {
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/ComponentDetailActivity.kt:104]: No need to react on the keys and we don't show indicator anymore

File: app/src/main/java/net/skyscanner/backpack/demo/data/SharedPreferences.kt [hunk 4]
```diff
@@ -18,38 +18,14 @@
...
 import android.content.Context
 import net.skyscanner.backpack.demo.R
 
 class SharedPreferences {
 
   companion object {
-    val SHOULD_HIGHLIGHT = "highlight"
     val THEME = "theme"
 
-    fun shouldHighlight(context: Context): Boolean {
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/data/SharedPreferences.kt:31]: This is when the indication settings were saved as persistent app settings.

**Review by shaundon — APPROVED** [review]

**Pull request merged**